app/src/main/java/com/vultisig/wallet/ui/models/settings/CheckForUpdateViewModel.kt

```
@@ -0,0 +1,57 @@
+                BuildConfig.VERSION_CODE.toString(),
+            )
+        )
+    )
+
+    init {
+        checkUpdate()
+    }
+
+    private fun checkUpdate() {
+
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)